### ISSUE TITLE
DSpot Maven plugin misalignment with latest DSpot release 

### ISSUE THREAD

**Issue by jesus-gorronogoitia** [STAMP-project/dspot#697]
#### Characteristics
- **Issue Type**: feature
- **Reproducibility**: always
- **Severity**: feature
- **Tool/Service/Component**: DSpot Maven plugin
- **Execution Environment**: All
- **Reporter**: Jesús Gorroñogoitia<jesus.gorronogoitia@atos.net>

#### Description
Current latest DSpot Maven plugin available in Maven Central is 1.2.3, but latest release DSpot version is 2.0.0.
Who (team) is developing DSpot Maven? I presume this Maven plugin is not automatically updated (CI/CD) upon new releases of DSpot, and requires further human development, being this the reason of this misalignment between DSpot and its Maven version.

##### Properties file

```java
project=.
src=src/main/java/
testSrc=src/test/java/
```

##### Command Line / Options

```shell
java -jar target/dspot-USED-VERSION-jar-with-dependencies.jar \
-p myPropertiesFile.properties
```

#### [Help on issue template](https://github.com/STAMP-project/dspot/blob/master/.github/ISSUE_DOC.md) 
Preview to follow the link or open file .github/ISSUE_DOC.md

**Comment by danglotb** [comment]
Hello @jesus-gorronogoitia 

I don't understand. All the dspot's modules are released at the same time.

Please see the [dspot-maven-2.0.0](https://search.maven.org/artifact/eu.stamp-project/dspot-maven/2.0.0/maven-plugin) on maven central.

Best.

**Comment by jesus-gorronogoitia** [comment]
Bonjour @danglotb Thanks for clarifying. The reason I reported this issue is that I tried DSpot Maven with version 1.2.4 (which was released before version 2.0.0, if I remember it well) but Maven complained it was not available. I will check version 2.0.0 availability and update my proposal (pull request) for documentation update

**Comment by jesus-gorronogoitia** [comment]
I confirm DSpot 2.0.0 is available and working in Maven central. Clossing issue
